examples/video_dataset_loading/main.py

```diff
@@ -0,0 +1,109 @@
+from kale.loaddata.video_dataset import  VideoFrameDataset, imglist_totensor
+from torchvision import transforms
+import torch
+import matplotlib.pyplot as plt
+import os
+
+
+if __name__ == '__main__':
+    """
+    This demo uses the dummy dataset inside of the folder "demo_dataset".
+    It is structured just like a real dataset would need to be structured.
+    """
+    videos_root = os.path.join(os.getcwd(), 'demo_dataset')
+    annotation_file = os.path.join(videos_root, 'annotations.txt')
+
+    """ DEMO 1 WITHOUT IMAGE TRANSFORMS """
+    dataset = VideoFrameDataset(
```

Comment:
Will import kale.loaddata.video.frameset as VideoFrameDataset be better? I am not sure. We can discuss tomorrow.